application/back-office/WebApp/package.json

```diff
@@ -13,6 +13,7 @@
     "swagger": "openapi-typescript shared/lib/api/BackOffice.Api.json -o shared/lib/api/api.generated.d.ts",
     "publish": "npm run build && mkdir -p ../Api/publish/WebApp/dist/ && rsync -r ./dist/ ../Api/publish/WebApp/dist/"
   },
+  "dependencies": {},
```

Comment:
All downstream projected based on PlatformPlatform would need to be updated too, so unless this is making something fail, we should revert this.